package.json

```diff
@@ -2,11 +2,16 @@
   "name": "react-formbuilder",
   "version": "0.0.1",
   "description": "A relatively easy to use form builder (single or multipage), with error reporting and lots of room for customization ",
-  "main": "index.js",
+  "main": "dist/react-formbuilder.js",
   "directories": {
     "test": "test"
```

Comment:
it's probably a good idea to create an `.npmignore` and add this dir to it, as well as all the not-dist content, so that npmjs.com only has the dist file for requiring in?